gtfe/__init__.py

```diff
@@ -21,3 +16,5 @@ def get_epoch_for_datetime(datetime):
     rfc2822_formatted_datetime = datetime.strftime(RFC2822_FORMAT)
     time_tuple = emailutils.parsedate_tz(rfc2822_formatted_datetime)
     return emailutils.mktime_tz(time_tuple)
+def STUPID_NAME(STUPID_ARG):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this parameter "STUPID_ARG" to match the regular expression ^[_a-z][a-z0-9_]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=python%3AS117)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename function "STUPID_NAME" to match the regular expression ^[a-z_][a-z0-9_]{2,30}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=python%3AS1542)